p8e-proto-internal/src/main/proto/p8e/affiliate.proto

```diff
@@ -18,6 +18,7 @@ service AffiliateService {
 message AffiliateRegisterRequest {
     PrivateKey signing_private_key = 1;
     PrivateKey encryption_private_key = 2;
+    PublicKey auth_public_key = 3;
```

Comment:
Don't think we are using the register method in AffiliateGrpc for anything, but add this just in case we decide to, we don't forget about this.